.github/workflows/tpl-mirror-charts.yaml

```diff
@@ -0,0 +1,66 @@
+name: "Mirror Helm Charts"
+
+on:
+  workflow_call:
+    inputs:
+      registry-destination:
+        type: string
+        required: true
+        description: "Repository to pull Helm charts from"
+      target-branch:
+        type: string
+        description: "Branch to commit downloaded Helm charts"
+        default: "charts"
+
+permissions:
+  contents: write
+
+jobs:
+  push-charts-to-museum:
+    name: "Checkout git-store"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout repository branch"
+        uses: actions/checkout@v2
+        with:
+          ref: "${{ inputs.target-branch }}"
+          fetch-depth: 0
+
+      - name: "Install Helm cm-push"
+        run: |
+          cd /tmp
+          wget https://github.com/chartmuseum/helm-push/releases/download/v0.10.4/helm-push_0.10.4_linux_amd64.tar.gz
```

Comment:
Same as this here, a dynamic version could be dope (and maybe default it if not specified)